Doc/whatsnew/3.14.rst

```diff
@@ -581,7 +581,10 @@ multiprocessing
      ``d |= {'b': 2}`` for proxies of :class:`dict`.
 
   (Contributed by Roy Hyunjin Han for :gh:`103134`.)
-
+* Add support for shared :class:`set` objects via
```

Comment:
I know that the other entries don't have a new line between them, but it's better to be add one so that it's clearer to read. We can do a copy-edit afterwards (cc @hugovk)

```suggestion

* Add support for shared :class:`set` objects via
```